packages/carbon-graphs/src/js/controls/Graph/Graph.js

```diff
@@ -426,64 +426,63 @@ class Graph extends Construct {
      * @returns {Graph} - Graph instance
      */
   reflow(graphData) {
-    let position;
-    if (graphData && graphData.values) {
-      this.contentKeys.forEach((key, index) => {
-        if (key === graphData.key) position = index;
-      });
-      if (position >= 0) {
-        if (
-          this.content[position].type === 'Bar'
-                    && graphData.values.length > 0
-        ) {
-          this.config.axis.x.ticks.values = [];
-          graphData.values.forEach((v) => this.config.axis.x.ticks.values.push(v.x));
+    let position = -1;
+    if (graphData && graphData.panData && graphData.panData && !utils.isEmptyArray(graphData.panData)) {
```

Comment:
may i know why graphData.panData is repeated?

Reply:
Removed it here https://github.com/cerner/terra-graphs/pull/61/commits/65d50998890f6818b61061e24719bf041b726d52